src/plugins/Network.js

```diff
@@ -161,8 +163,13 @@ module.exports = class Network extends OverlayPlugin {
         const values = message.split(' ');
 
         if (values[0] === 'if') {
-            this.wifiInput.disabled = false;
-            this.mobileInput.disabled = false;
+            if (this.wifiInput) {
```

Comment:
Is it possible to use optional chaining?


```javascript
this.wifiInput?.disabled = false
```
?

Depends on what version of javascript is supported. More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Actually it looks like this isn't possible after all:
https://github.com/tc39/proposal-optional-chaining#not-supported

> The following is not supported, although it has some use cases; see [Issue #18](https://github.com/tc39/proposal-optional-chaining/issues/18) for discussion:
> * optional property assignment: `a?.b = c`